octodns/provider/constellix.py

```diff
@@ -328,16 +813,19 @@ def _params_for_multiple(self, record):
     # there is an A record before creating NS
     _params_for_NS = _params_for_multiple
 
-    def _params_for_single(self, record):
+    @staticmethod
+    def _params_for_single(record):
         yield {
             'name': record.name,
             'ttl': record.ttl,
             'host': record.value,
         }
 
     _params_for_CNAME = _params_for_single
+    _params_for_PTR = _params_for_single
```

Comment:
`_params_for_PTR` are not the same as `_params_for_single`, the previous `_params_for_ALIAS` were correct

```
Traceback (most recent call last):
  File "/Users/yzguy/Projects/octodns/env/bin/octodns-sync", line 11, in <module>
    load_entry_point('octodns==0.9.8', 'console_scripts', 'octodns-sync')()
  File "build/bdist.macosx-10.14-x86_64/egg/octodns/cmds/sync.py", line 39, in main
  File "build/bdist.macosx-10.14-x86_64/egg/octodns/manager.py", line 330, in sync
  File "build/bdist.macosx-10.14-x86_64/egg/octodns/provider/base.py", line 93, in apply
  File "build/bdist.macosx-10.14-x86_64/egg/octodns/provider/constellix.py", line 651, in _apply
  File "build/bdist.macosx-10.14-x86_64/egg/octodns/provider/constellix.py", line 589, in _apply_create
  File "build/bdist.macosx-10.14-x86_64/egg/octodns/provider/constellix.py", line 246, in record_create
  File "build/bdist.macosx-10.14-x86_64/egg/octodns/provider/constellix.py", line 123, in _request
octodns.provider.constellix.ConstellixClientBadRequest:
  - Standard should have atleast 1 active value
```